aestream/input.py

```diff
@@ -0,0 +1,23 @@
+from aestream import aestream_ext as ext
+
+try:
+    import torch
+    USE_TORCH = True
+except ImportError:
+    USE_TORCH = False
+
+class FileInput(ext.FileInput):
```

Comment:
Fine for now but when we implment the new API we will change this. We should take examples from other libraries and create concepts from them in the api planning doc file

Reply:
Do you mean the Python API will be changed? Or the torch import?

Reply:
Python API